### PR TITLE
Allow custom create callback.

### PR DESCRIPTION
**Pull request by haringsrob**

### CODE REVIEW

File: resources/views/login-error.blade.php [hunk 1]
```diff
@@ -4,7 +4,7 @@
       {{-- <svg class="h-5 w-5 text-red-400" viewBox="0 0 20 20" fill="currentColor" aria-hidden="true">
         <path fill-rule="evenodd" d="M10 18a8 8 0 100-16 8 8 0 000 16zM8.28 7.22a.75.75 0 00-1.06 1.06L8.94 10l-1.72 1.72a.75.75 0 101.06 1.06L10 11.06l1.72 1.72a.75.75 0 101.06-1.06L11.06 10l1.72-1.72a.75.75 0 00-1.06-1.06L10 8.94 8.28 7.22z" clip-rule="evenodd" />
       </svg> --}}
-	  <x-icon name="far-circle-x" class='w-8 text-red-400' />
+{{--	  <x-icon name="far-circle-x" class='w-8 text-red-400' />--}}
```
Comment (haringsrob) [inline resources/views/login-error.blade.php:7]: Not everyone has this icon set.

Comment (chrisreedio) [reply resources/views/login-error.blade.php:7]: Good catch. 

I was looking to wrap up the beta and get it released alongside Laravel 11 support.

I'll double check and make sure any font awesome icons have been removed.

Comment (mstfkhazaal) [reply resources/views/login-error.blade.php:7]: any update?

Comment (chrisreedio) [reply resources/views/login-error.blade.php:7]: This was fixed in v3.6.1 by #49

Comment (chrisreedio) [reply resources/views/login-error.blade.php:7]: Can you please remove this from the PR?

**Review by chrisreedio — CHANGES_REQUESTED** [review]
Since the pro icon was already removed, please merge the latest commits into this and it should clear up the icon issue.

**Comment by cheesegrits** [comment]
@haringsrob @chrisreedio hey guys, I was literally just sitting down to code up (almost) this exact PR.

My use case is I need to be able to abort an account creation, which would almost be handled by this PR in that I could just throw an AbortedLoginException in my createAccountUsing callback.

I think the only difference would be I would probably need to allow for specifying a redirect URL.

Any chance this PR can be merged so I can start using it, and if necessary PR a redirect URL change?

**Comment by chrisreedio** [comment]
> @haringsrob @chrisreedio hey guys, I was literally just sitting down to code up (almost) this exact PR.
> 
> My use case is I need to be able to abort an account creation, which would almost be handled by this PR in that I could just throw an AbortedLoginException in my createAccountUsing callback.
> 
> I think the only difference would be I would probably need to allow for specifying a redirect URL.
> 
> Any chance this PR can be merged so I can start using it, and if necessary PR a redirect URL change?

Thanks for the nudge to get me moving on this again.

Went with a first party PR on this to resolve the need due to no movement on this. 

See #59 for more.

**Pull request closed without merging**